packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -71,6 +72,9 @@ function addSearchTerm(term) {
 	// @ts-ignore
 	inputElement.value?.$el.focus();
 }
+const toggleSearchByExample = () => {
+	emit('toggle-search-by-example');
```

Comment:
If this is the only thing this method does, you can just emit directly from `@click` instead of having a function that just calls another function

Reply:
sure but this is temporary placeholder code that is not being used at this time and will likely be refactored later!